bigchaindb/consensus/__init__.py

```diff
@@ -0,0 +1,43 @@
+# TODO: no real reason to use abc yet, but later we can enforce inheritance from
+#       this class when loading plugins if that's desirable.
+# from abc import ABCMeta
+
+class AbstractConsensusRules:
+
+    # TODO: rather than having plugin-authors inherit and override,
+    #       it'd be cleaner to make a `transactionrule` decorator and etc
```

Comment:
Agree, both `validate_transaction` and `validate_block` can be simple functions instead of `classmethod`s. But we can do it later.